utils/libostree/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2024 Muhamad Aji Wibisono <muhamadajiw71@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libostree
+PKG_VERSION:=2024.8
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/ostreedev/ostree/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=e2136e116669f114ff7349f12e299f63c0bece733451649414a8dcd85b863c3f
+
+PKG_BUILD_DEPENDS:=glib2 libarchive libsoup3 fuse3
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_LICENSE:=GPL-2.0
```

Comment:
Deprecated SPDX License identifier.

Reply:
Ah right, it should've been GPL-2.0-only